packages/p2p/src/pages/buy-sell/buy-sell-form.jsx

```diff
@@ -174,9 +175,9 @@ const BuySellForm = props => {
 
                     return (
                         <Form noValidate>
-                            <div className='buy-sell__modal-content'>
-                                <div className='buy-sell__modal-field-wrapper'>
-                                    <div className='buy-sell__modal-field'>
+                            <div className='buy-sell-form-content'>
```

Comment:
```suggestion
                            <div className='buy-sell-form__content'>
```

could we do this for the other className's as well here? we could do __